src/test/java/FlattenerTest.java

```diff
@@ -20,8 +20,8 @@ public void shouldFlattenFields() throws IOException, DocumentException {
         //Check that the PdfFormField "name" is present in the document.
         Assert.assertTrue(TestUtils.checkDocumentIsFlat(fileToCheck));
 
-        final ByteArrayOutputStream fileAfterFlattened = LambdaFlattener.execute(fileToFlattenSpec);
-        final byte[] fileFlattened = fileAfterFlattened.toByteArray();
+        final byte[] fileAfterFlattened = LambdaFlattener.execute(fileToFlattenSpec);
+        final byte[] fileFlattened = fileAfterFlattened;
```

Comment:
not necessary anymore

Reply:
This is getting embarrassing :). Thanks for your patience Martin. I removed the redundant code.

Reply:
No need to apologize, this is not a small change, it's easy to overlook little things :)